src/main/scala/chisel3/experimental/inlinetest/InlineTest.scala

```diff
@@ -93,7 +93,8 @@ object TestHarnessGenerator {
   *  @tparam TestResult the type returned from each test body generator, typically
   *  hardware indicating completion and/or exit code to the testharness.
   */
-trait HasTests[M <: RawModule] { module: M =>
+trait HasTests { module: RawModule =>
+  type M = module.type
```

Comment:
Given that this is now defined, can this be used to replace existing `module.type` in this trait?